README.md

```diff
@@ -3,17 +3,21 @@ A framework for autonomous economic agent (AEA) development
 
 ## Get started
 
-- Create and launch a virtual environment with Python 3.7 (any Python `>=` 3.6 works):
+1. Create and launch a clean virtual environment with Python 3.7 (any Python `>=` 3.6 works):
 
       pipenv --python 3.7 && pipenv shell
 
-- Install the package from [pypi](https://pypi.org/project/aea/):
+2. Install the package from [PyPI](https://pypi.org/project/aea/):
 
 
       pip install aea[all]
 
 
-- Then, build your agent as described in the [docs](https://fetchai.github.io/agents-aea/).
+3. Then, build your agent as described in the [docs](https://fetchai.github.io/agents-aea/).
+
+<p align="center">
```

Comment:
Adding this here both as a visual element (we can put something better in the future) and also as a divider..Makes it less likely people jump straight on to the next section (which we don't want them to unless they have interest in contributing)